.github/workflows/bump_metallb.yml

```diff
@@ -0,0 +1,60 @@
+name: Periodic MetalLB Bump
+on:
+  schedule:
+    - cron: "0 7,19 * * *"
+
+jobs:
+  bump-metallb-and-create-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get current metallb commit hash
+        run: source hack/common.sh
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          repository: metallb/metallb
+          path: metallb
+      - name: Get old commit info
+        id: old-commit-info
+        run: |
+          cd metallb
+          echo "::set-output name=commit_sha::${{ env.METALLB_COMMIT_ID:0:7 }}"
+          echo "::set-output name=commit_title::$(git log -1 --pretty=%s ${{ env.METALLB_COMMIT_ID }})"
+          echo "::set-output name=commit_date::$(git log -1 --pretty=%cd ${{ env.METALLB_COMMIT_ID }})"
+      - name: Get new commit info
+        id: new-commit-info
+        run: |
+          echo "::set-output name=commit_sha::$(git rev-parse --short=7 HEAD)"
+          echo "::set-output name=commit_title::$(git log -1 --pretty=%s)"
+          echo "::set-output name=commit_date::$(git log -1 --pretty=%cd)"
+          cd ..
+      - name: Check if there are changes
+        id: check-for-changes
+        run: |
+          if steps.old-commit-info.outputs.commit_sha == steps.new-commit-info.outputs.commit_sha; then
+            echo "::set-output name=has_changes::false"
+          else
+            echo "::set-output name=has_changes::true"
+          fi
+      - name: Bump metallb
+        if: ${{ steps.check-for-changes.outputs.has_changes == 'true' }}
+        run: |
+          make bump-metallb
+      - name: Create pull request
+        if: ${{ steps.check-for-changes.outputs.has_changes == 'true'}}
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: |
+            Bump MetalLB
+
+            This commit bumps MetalLB from `${{ steps.old-commit-info.outputs.commit_sha }}` to `${{ steps.new-commit-info.outputs.commit_sha }}`.
```

Comment:
Is there a way to close any existing already opened pr?
Thinking about a scenario where we bump  metallb multiple times but we wait to bump the operator.
Ideally, only the most recent one should be open

Reply:
Yes, that could be nice. Need to figure out how to query the open PR and check if it should be kept or closed.

Reply:
the workflow will periodically override old PRs with the latest metallb. so I think it could work for us.